cypress/integration/move.spec.js

```diff
@@ -16,9 +16,144 @@ describe("Move flow", () => {
     TEST_REPO_SUBFOLDER_NAME
   )
 
-  // TODO: after refactor, add tests for Workspace
-  // describe("Move pages out of Workspace", () => {
-  // })
+  describe("Move pages out of Workspace", () => {
+    const TITLE_WORKSPACE_TO_FOLDER = "Move from Workspace to folder"
+    const FILENAME_WORKSPACE_TO_FOLDER = titleToPageFileName(
+      TITLE_WORKSPACE_TO_FOLDER
+    )
+
+    const TITLE_WORKSPACE_TO_SUBFOLDER = "Move from Workspace to subfolder"
+    const FILENAME_WORKSPACE_TO_SUBFOLDER = titleToPageFileName(
+      TITLE_WORKSPACE_TO_SUBFOLDER
+    )
+
+    beforeEach(() => {
+      cy.setCookie(COOKIE_NAME, COOKIE_VALUE)
+      window.localStorage.setItem("userId", "test")
+      cy.visit(`${CMS_BASEURL}/sites/${TEST_REPO_NAME}/workspace`)
+    })
+
+    it("Should be able to navigate from Workspace to subfolder back to Workspace via MoveModal buttons", () => {
+      cy.contains(FILENAME_WORKSPACE_TO_FOLDER).should("exist")
+      cy.get(
+        `button[id^="folderItem-dropdown-${FILENAME_WORKSPACE_TO_FOLDER}"]`
```

Comment:
Should this be `button[id^="pageCard-dropdown-${FILENAME_WORKSPACE_TO_FOLDER}"]` instead? Few other instances of this in the move test

Reply:
good catch - fixed in c50f7bd